contrib/test_jenkins.sh

```diff
@@ -762,42 +753,24 @@ run_ucx_perftest() {
 
 		echo "==== Running ucx_perf with cuda memory===="
 
-		for tls in $tls_list
+		for gdr in $gdr_options
 		do
-			for memtype_cache in y n
```

Comment:
why don't test it anymore?

Reply:
maybe it is worth having a reduced number of tests that are run for GPU variants?

Reply:
ic, but this reduces the coverage

Reply:
> ic, but this reduces the coverage

Yes, I agree.
I suggest reducing the number of tests (tag_bw/tag_lat and etc) or its iterations instead of not checking different memtype_cache/tls variants.

Reply:
in general i'd go with longer test execution time but better coverage

Reply:
IMO no need to test with memtype cache disabled, since the code flows for memtype cache enabled already include memory detection. it's just not done every time like with memtype cache disabled.

Reply:
ok, restored memtype cache on/off
since gdrcopy on/off is tested by test_ucp_tag_mem_type gtest